src/app/d2-vendors/Vendor.tsx

```diff
@@ -110,7 +71,7 @@ export default class Vendor extends React.Component<Props, State> {
       <div className='vendor-char-items'>
         <div className='title'>
           <div className="collapse-handle">
-            {dropLinkRelevant && <a target="_blank" rel="noopener" href="https://vendorengrams.xyz/"><img className={vendorEngramClass} src={vendorEngramSvg} title={t(vendorLinkTitle)} /></a>}
+            {vendorEngramDrops.length > 0 && powerLevelMatters(basePowerLevel) && <a target="_blank" rel="noopener" href="https://vendorengrams.xyz/"><img className={vendorEngramClass} src={vendorEngramSvg} title={t(vendorLinkTitle)} /></a>}
```

Comment:
This test can just be `vendorEngramDrops.length && powerLevelMatters(basePowerLevel)`, right?

Reply:
React will skip rendering `false` or `null` or `undefined`. But `0 && true` evaluates to `0`, which will actually display a "0" on the page. Alternately I could do `Boolean(vendorEngramDrops.length) && powerLevelMatters(basePowerLevel)`.

Reply:
Nah, what's there is fine. I didn't realize that it had that particular rendering quirk - that's SGA!